xml/admin_administration.xml

```diff
@@ -75,13 +95,30 @@
   </note>
 
   <sect2 xml:id="transactional.updates.installation">
-   <title>Manual Installation of Updates</title>
+   <title>Manually Applying Updates</title>
+   
    <para>
     After the <command>transactional-update</command> script has run on all
     nodes, &dashboard; displays any nodes in your cluster running outdated
-    software. &dashboard; then enables you to update your cluster directly.
+    software. The updates are only applied after a refresh. For this purpose,
+    &dashboard; enables you to update your cluster directly.
     Follow the next procedure to update your cluster.
    </para>
+   
+   <important>
+    <title>Manual Transactional Update</title>
+    <para>
+     If you have used <command>transactional-update shell</command> to manually 
+     create a snapshot and modify transactional updates, make sure that the 
+     node is rebooted as soon as possible after you are finished. If the 
+     automated transactional update script runs before you reboot, your manual 
+     changes will be overwritten by the automatically created snapshot. Moreover, 
+     if you perform another manual installation before a reboot, only the last 
+     snapshot will be used and all prior changes are lost. Either perform all 
+     updates in one session or reboot the node(s) between each step.
+  </para>
+ </important>
+ 
```

Comment:
Above we write, the customer should never do a hard reboot, now we leave him alone and don't tell him, how to trigger the reboot correct.
Else: the title "Manual Transactional Update" does not fit to the description, which describes "transactional-update shell", not how to manual update the system.
Is there really a valid usecase for an admin to know and use this option?

Reply:
I wouldn't know if there is a use case for manual updates via shell.
@ajaeger Is that supposed to be supported or should we hide this altogether?

@thkukuk The proper way to update the system is described in "Updating the Cluster with Velum." that doesnt' cover what you're asking?